esp/services/ws_dfu/ws_dfuService.hpp

```diff
@@ -95,6 +95,27 @@ class CThorNodeGroupCache: public SuperHashTableOf<CThorNodeGroup, const char>
     CThorNodeGroup *lookup(const char* groupName, unsigned timeOutMinutes);
 };
 
+struct DFUReadAccessRequest
+{
+    const char *logicalName = nullptr;
+    const char *clusterName = nullptr;
+    IUserDescriptor *udesc  = nullptr;
+    CSecAccessType accessType;
+    unsigned expiry = 1;
```

Comment:
is this expiry in secs or mins? Other defaults are 60 secs

Reply:
DFUReadAccessRequest DFUReadAccessResponse structs are not used in this branch, and will be deleted.